lib/jelix-admin-modules/jauthdb_admin/templates/crud_list.tpl

```diff
@@ -5,8 +5,8 @@
 <form action="{formurl 'jauthdb_admin~default:view'}" method="get">
     <div>
         <label for="search-login">{@jauthdb_admin~crud.title.view@}</label>
-        <input id="search-login" name="j_user_login">
-        <button type="submit" onsubmit="test();">{@jauthdb_admin~crud.link.view.record@}</button>
+        <input id="search-login" name="j_user_login" link_autocomplete="{jurl 'jauthdb_admin~default:autocomplete'}">
```

Comment:
You should use a 'data' like attribute, like `data-link-autocomplete`. See [HTML doc about them](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*) to read the corresponding JS API to retrieve `data-*` attributes (or the jquery API). Custom attributes that are not `data-*` attributes are not allowed by the HTML5 specification.